assimilation_code/modules/utilities/ensemble_manager_mod.html

```diff
@@ -118,26 +118,17 @@ <H2>NAMELIST</H2>
 
 <TR><TD>layout</TD>
     <TD>integer</TD>
-    <TD>Determines the logical process (PE) layout across MPI tasks.
+    <TD>Determines the process (pe) layout on MPI tasks.
      1 is PE = MPI task. 2 is a round-robin layout around the nodes. Layout 2
      results in a more even usage of memory across nodes. This may allow you to run
      with a larger state vector without hitting the memory limit of the node.
      It may give a slight (5%) increase in performance, but this is machine dependent.
-     It has no effect on serial runs.
+     It has no effect on serial runs of filter.
 </TD></TR>
 
 <TR><TD>tasks_per_node</TD>
     <TD>integer</TD>
-    <TD>The number of MPI tasks per hardware node is generally fixed when a batch job 
-       is submitted. This namelist item tells the ensemble manager what the user 
-       selected at that time.  Once a program is running the code has no control 
-       to change how MPI tasks are assigned to physical CPUs.  
-       This number is used only if layout = 2, and
-       it allows the code spread high-memory-use PEs to different hardware nodes by
-       assigning them in a round-robin order.  The job will still run if this
-       number does not match the real "tasks_per_node" at the hardware level,
-       but it may run out of memory if the mismatch causes multiple high-memory-use
-       tasks to be run on the same node.
+    <TD>The number of tasks per node.  This is only used if layout = 2. 
```

Comment:
tim - is this a case where the public master was more recent than the development?  this longer description seems helpful and unless someone removed it it seems like you may be losing changes you want to keep.